CONTRIBUTORS.md

```
@@ -615,4 +615,8 @@ The following is a list of contributors to the .NET **nanoFramework** project (a
     <td><img src="https://github.com/dcyonce.png?size=50" height="50" width="50" ></td>
     <td><a href="https://github.com/dcyonce">Don Yonce</a></td>
   </tr>
+    <tr>
+    <td><img src="https://github.com/benyuz.png?size=50" height="50" width="50" ></td>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)